UKHomeOffice/cop-react-components
# COP-10002: Fix vulnerability issues

### Description
Work done by Ali Khattab and Keenan Batley to get everything compiling and playing nicely with the `yarn audit`, such that the only vulnerability issues are non-production ones (these will be addressed separately).

https://support.cop.homeoffice.gov.uk/browse/COP-10002

### To test
The build should work, Storybook should compile and run just fine (and look correct), and the following command should bring back no vulnerabilities:
`yarn audit --groups "dependencies"`
